### PR TITLE
Monsters of the Multiverse

### PR DESCRIPTION
**Pull request by AngryGuineaPig**

### CODE REVIEW

File: supplements/monsters-of-the-multiverse/race-aarakocra.xml [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+			<p class="indent">Intelligence, Wisdom, or Charisma is your spellcasting ability for it when you cast <i>gust of wind</i> with this trait (choose when you select this race).</p>
+		</description>
+		<sheet display="false" />
+		<setters>
+		</setters>
+		<rules>
+			<select type="Ability Score Improvement" name="Ability Score Improvement Option (Aarakocra)" supports="Custom Ability Score Improvement Option" />
+			<stat name="innate speed" value="30" bonus="base" />
+			<grant type="Size" id="ID_SIZE_MEDIUM" />
+
```
Comment (AngryGuineaPig) [inline supplements/monsters-of-the-multiverse/race-aarakocra.xml:31]: Missing Creature Type Grant

File: supplements/monsters-of-the-multiverse/race-aasimar.xml [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+		</description>
+		<sheet display="false" />
+		<setters>
+		</setters>
+		<rules>
+			<select type="Ability Score Improvement" name="Ability Score Improvement Option (Aasimar)" supports="Custom Ability Score Improvement Option" />
+			<stat name="innate speed" value="30" bonus="base" />
+			<select type="Size" name="Size (Aasimar)" supports="ID_SIZE_SMALL||ID_SIZE_MEDIUM" />
+			<grant type="Vision" id="ID_VISION_DARKVISION" />
+
```
Comment (AngryGuineaPig) [inline supplements/monsters-of-the-multiverse/race-aasimar.xml:48]: Missing Creature Type Grant

File: supplements/monsters-of-the-multiverse/race-genasi.xml [hunk 3]
```diff
@@ -0,0 +1,432 @@
...
+			<p class="indent"><strong><em>Creature Type.</em></strong> You are a Humanoid.</p>
+			<p class="indent"><strong><em>Size.</em></strong> You are Medium or Small. You choose the size when you select this race.</p>
+			<p class="indent"><strong><em>Speed.</em></strong> Your walking speed is 30 feet.</p>
+			<p class="indent"><strong><em>Darkvision.</em></strong> You can see in dim light within 60 feet of you as if it were bright light, and in darkness as if it were dim light. You discern colors in that darkness only as shades of gray.</p>
+			<p class="indent"><strong><em>Subrace.</em></strong> Four major subraces of genasi are found among the worlds of D&amp;D: air genasi, earth genasi, fire genasi, and water genasi. Choose one of these subraces.</p>
+		</description>
+		<sheet display="false" />
+		<rules>
+			<select type="Ability Score Improvement" name="Ability Score Improvement Option (Genasi)" supports="Custom Ability Score Improvement Option" />
+			<select type="Size" name="Size (Genasi)" supports="ID_SIZE_SMALL||ID_SIZE_MEDIUM" />
```
Comment (AngryGuineaPig) [inline supplements/monsters-of-the-multiverse/race-genasi.xml:25]: needs to be type="Racial Trait"

Comment (kgirtxd) [reply supplements/monsters-of-the-multiverse/race-genasi.xml]: Fixed in [9b0b613](https://github.com/AuroraLegacy/elements/pull/119/commits/9b0b613d832291909c6d2fd9d553b65b82637b5e)

**Pull request closed without merging**